instrumentation/kafka/kafka-connect-2.6/testing/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.javaagent-testing")
+}
+
+dependencies {
+  api(project(":testing-common"))
+
+  implementation("org.apache.kafka:kafka-clients:0.11.0.0")
+
+  implementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-common-0.11:library"))
+  implementation(project(":instrumentation:kafka:kafka-connect-2.6:javaagent"))
+  implementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-0.11:javaagent"))
+
+  testImplementation("org.apache.kafka:connect-runtime:3.6.1")
+  testImplementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-0.11:testing"))
+
+  implementation("org.testcontainers:postgresql:1.21.3") // For PostgreSQLContainer
```

Comment:
since this module only has tests it is strange to see a mix of `implementation` and `testImplementation` dependencies